scripts/boxes2inkscape

```diff
@@ -140,9 +140,6 @@ class Boxes2INX:
 
     def writeAllINX(self, path):
         for name, box in self.boxes.items():

```

Comment:
I doubt we can do this. Well, Inkscape users could type in the whole layout but that's not really practical.

Reply:
> I doubt we can do this. Well, Inkscape users could type in the whole layout but that's not really practical.

I've never used the inkscape plugin, so I don't know how it would work.

> I guess we can't get rid of the 2 step process for the command line interface - if we want to keep supporting that. Otoh that's what the currently worked on smoke test is using (and also having trouble with the TrayLayout generator).

But for the command-line, yes it would still be a 2 step process, but I'm not sure it needs to have special code handling.  Could it work like:
```
$ boxes TrayLayoutGenerator --x 50 --y100 --nx 6 --ny 4  -o mylayout.txt
# Edit mylayout.txt
$ boxes TrayLayoutGenerator --input mylayout.txt  -o mybox.svg
```

perhaps something similar for inkscape?  Having specifically named generators in the scripts directory feels wrong to me. :-)